src/uct/api/uct.h

```diff
@@ -1958,9 +1959,10 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
  *    capability flag. It may be obtained by @ref uct_iface_query .
  * -# Connect to a remote socket address: If @ref uct_ep_params_t::sockaddr is
  *    set, this will create an endpoint that is connected to a remote socket.
- *    This requires that @ref uct_ep_params::cm or @ref uct_ep_params::iface
- *    with @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability flag is set.
- *    Interface capabilities may be obtained by @ref uct_iface_query .*
+ *    This requires that @ref uct_ep_params::cm or @ref uct_ep_params::iface has
+ *    to be set. Interface has to support
+ *    @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability flag. Interface
+ *    capabilities may be obtained by @ref uct_iface_query.*
```

Comment:
`This requires that either @ref uct_ep_params::cm, or @ref uct_ep_params::iface will be set. In the latter case, the interface has to support @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability flag, which can be checked by calling @ref uct_iface_query.`